socorro/processor/rules/general.py

```diff
@@ -3,19 +3,20 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 from glom import glom
+import markus
 
 from socorro.processor.rules.base import Rule
 
 
+METRICS = markus.get_metrics("processor")
+
+
 class DeNullRule(Rule):
     """Removes nulls from keys and values
 
     Sometimes crash reports come in with junk data. This removes the egregious
     junk that causes downstream processing and storage problems.
 
-    NOTE(willkg): While this changes the raw crash, that raw crash should never
-    get saved back to storage, so this doesn't overwrite the original material.
```

Comment:
This isn't needed anymore. I nixed `save_raw_and_processed` and changed the contract so that `save_processed_crash` never saves raw crash data.